code/modules/martial_arts/cqc.dm

```diff
@@ -44,22 +53,23 @@
 /datum/martial_art/cqc/proc/Slam(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	if(!can_use(A))
 		return FALSE
-	if(!D.stat || !D.weakened)
+	if(!D.weakened && !D.resting && !D.lying)
```

Comment:
The logic of this is just wrong; it's meant to fail if they're on the ground, at all. Repeat slams are super OP.